app/assets/javascripts/social_networking/controllers/goal-controller.js

```diff
@@ -154,6 +155,13 @@
     this._goalTool.setFilter(type);
   };
 
+  GoalCtrl.prototype.showCharLimit = function(inputTag) {
```

Comment:
This seems like a valid placement to me. Its not creating new service logic, just using a service library. It also is directly manipulating elements on the view that utilizes this controller, so I don't think it'd be necessary to abstract it further away.